zulip_bots/zulip_bots/bots/susi/susi.py

```diff
@@ -41,9 +41,39 @@ def handle_message(self, message: Dict[str, str], bot_handler: Any) -> None:
             return
         reply = requests.get("https://api.susi.ai/susi/chat.json", params=dict(q=msg))
         try:
-            answer = reply.json()['answers'][0]['actions'][0]['expression']
-        except Exception:
+            data = reply.json()
+            actions = data['answers'][0]['actions']
+            answer = ""
+            for action in actions:
+                if action['type'] == 'answer':
+                    answer += action['expression'] + '\n'
+                elif action['type'] == 'anchor':
+                    text = action['text']
+                    link = action['link']
+                    answer += '[' + text + '](' + link+')' + '\n'
+                elif action['type'] == 'table':
+                    tableData = data['answers'][0]['data']
+                    msg = ""
+                    columns = list(action['columns'].keys())
+                    columnsData = list(action['columns'].values())
+                    # using markdown make a table
+                    for heading in columnsData:
+                        answer += "| " + heading + " "
+                    answer += "\n"
+                    for heading in columns:
+                        answer += "|" + "------" + " "
+                    answer += "\n"
+                    for item in (tableData):
+                        if (item[columns[0]]):
+                            column1 = item[columns[0]] if (type(item[columns[0]]) is str) else item[columns[0]][0]
```

Comment:
hmm. the variable names seems to not make a lot of sense. Its minor, but it would be great to know what it is rather than a,b,c, a1, b1, c1 etc